src/licensedcode/data/licenses/rh-eula-lgpl.LICENSE

```diff
@@ -1,11 +1,13 @@
 ---
 key: rh-eula-lgpl
-short_name: Red Hat EULA JBoss Enterprise Middleware License
-name: Red Hat EULA JBoss Enterprise Middleware License
+short_name: Red Hat LGPL-Based EULA
+name: Red Hat LGPL-Based End User License Agreement
 category: Copyleft Limited
 owner: Red Hat
-homepage_url: https://www.redhat.com/f/pdf/licenses/GLOBAL_EULA_JBoss_English_20101110.pdf
+homepage_url: https://www.redhat.com/en/about/eulas
```

Comment:
This lgpl EULA does not look the same as JBoss one:
- https://www.redhat.com/licenses/Red_Hat_LGPL-Based_EULA_20191118.pdf
- https://www.redhat.com/f/pdf/licenses/GLOBAL_EULA_JBoss_English_20101110.pdf

these are super similar but are they really the same? We want to keep both text URLs in all cases and we may need some rules?